scripts/corr2_sensor_servlet.py

```diff
@@ -100,7 +100,7 @@ def on_shutdown(ioloop, server):
         raise RuntimeError('Received nonsensical log level %s' % args.loglevel)
 
     # set up the logger
-    root_logger = logging.getLogger()
+    root_logger = logging.getLogger('corr2.sensors')
```

Comment:
Perhaps rename the variable to 'corr2_sensors_logger' or just 'logger' since 'root_logger' has a specific meaning, i.e. "The root python logger" which is what you get when you call logging.getLogger() without parameters.